snippets/abbreviations.md

```diff
@@ -12,8 +12,13 @@
 *[BAU]: Business as Usual
 *[SOP]: Standard Operating Procedures
 *[CMS]: Case Management System
+*[CIN1]: Case management system Integration environment One
```

Comment:
There **isn't** a CIN1
There **is** a CIN5
there is also a PT for performance testing
PCMS = production CMS
LCMS = ??
MO = Model Office CMS